tests/test_app/Plugin/TestPlugin/src/Database/Driver/TestDriver.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright 2005-2011, Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright 2005-2011, Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.2.7
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace TestPlugin\Database\Driver;
+
+use Cake\Database\Driver\Sqlite;
+
+class TestDriver extends Sqlite
+{
+
```

Comment:
This is a suggestment for you then to write a new sniffer if you are running out of ideas :)
As in this example it just affects test files, in other cases it might effect more than that.